packages/qwik/src/optimizer/src/plugins/plugin.ts

```diff
@@ -428,7 +428,7 @@ export function createPlugin(optimizerOptions: OptimizerOptions = {}) {
         };
       }
 
-      const firstInput = Object.values(opts.input)[0];
+      const firstInput = Object.values(opts.input || [])[0];
```

Comment:
`const firstInput = opts.input && Object.values(opts.input)[0];`

This way you don't create objects for no reason